README.md

```diff
@@ -1 +1,7 @@
-Factory+ Documentation
\ No newline at end of file
+# Factory+ Documentation
+
+## Getting Started
+To run the Docusaurus server locally, run the following command from the root directory of this repository:
+```bash
+yarn && yarn start
```

Comment:
```suggestion
npm i
yarn start
```

Reply:
We don't mix `yarn` and `npm`. `yarn` is the same as `npm install`